fix-point/app/Main.hs

```diff
@@ -20,12 +21,8 @@ main = bootstrap =<< execParser opts
 
 bootstrap :: Options -> IO ()
 bootstrap (Options project pref source) = do
-    let prefix = case pref of
-                     (Just s) -> s
-                     Nothing  -> upperHead project
-    let sourceDir = case source of
-                        (Just s) -> s
-                        Nothing  -> "three-layer"
+    let prefix = maybe project upperHead pref
```

Comment:
Arguments to `maybe` are passed in wrong way. Now, as I can see, it's better to use `fromMaybe` here as well.